cpp/data/zotero_cgi/index.html

```diff
@@ -179,12 +184,9 @@ <h1>Zotero Harvesting</h1>
                                         <option{IF selected_output_format_id == output_format_ids} selected="selected"{ENDIF}>{output_format_ids}</option>
                                     {ENDLOOP}
                                 </select>
+                                MARC21 and MARCXML are custom. All others are generated by Zotero (JSON = native Zotero API format).
```

Comment:
This is no longer accurate - The harvester only supports MARC-XML as its output. In fact, we can disable the output format dropdown on the web interface.